pkg/resourcecreator/ingress/ingress.go

```diff
@@ -155,7 +173,11 @@ func nginxIngresses(source Source, cfg Config) ([]*networkingv1.Ingress, error)
 		ingressClass := util.ResolveIngressClass(rule.Host, cfg.GetGatewayMappings())
 
 		if ingressClass == nil {
-			return nil, fmt.Errorf("domain '%s' is not supported", rule.Host)
+			return nil,
+				fmt.Errorf("domain '%s' is not supported in this cluster. Your domain is '.%s', try one of these '%v'",
```

Comment:
Kunne `this cluster` sagt hvilket cluster man er i?

Reply:
Forsåvidt, men den er ikke uten videre tilgjengelig der?

Reply:
Hvis det ikke er tilgjengelig så ville jeg bare droppet det.

Reply:
Joda, hvilket cluster man kjører i er en del av basiskonfigurasjonen til Naiserator og skal være lett tilgjengelig gjennom config-objektet e.l.

Eller så kan man lage det.

Reply:
> Kunne this cluster sagt hvilket cluster man er i?

It doesn't matter. You get a list of valid domains that indicate this either way. 
If you get a bunch of dev.nav.no you know where you're at.

Reply:
f.eks?
> "This cluster supports these domains: [...]"

Reply:
Fix'd: 2ce586fa